content/en/getting-started/installation.md

```diff
@@ -292,7 +342,30 @@ $ helm upgrade --install localstack localstack-repo/localstack
 
 The Helm charts are not maintained in the main repository, but in a [separate one](https://github.com/localstack/helm-charts).
 
-### Troubleshooting
+
+## Updating
+
+The LocalStack CLI allows you to easily update the different components of LocalStack.
+To check the various options available for updating, run:
+{{< command >}}
+$ localstack update --help
+Usage: localstack update [OPTIONS] COMMAND [ARGS]...
+
+  Update different LocalStack components.
+
+Options:
+  -h, --help  Show this message and exit.
+
+Commands:
+  all             Update all LocalStack components
+  docker-images   Update docker images LocalStack depends on
+  localstack-cli  Update LocalStack CLI
```

Comment:
tiny nit: maybe simply `localstack update cli`? (no strong preference, though.. feel free to leave as-is)

Reply:
I was thinking the same thing, but I wanted to avoid renaming the command to make sure that we don't introduce any breaking changes (the command has been introduced quite a while ago)...
However, I think when we further move towards binary packages, we could deprecate it at some point and remove it (because binaries should not update themselves, this should be done by the package manager).